nx/test/nx/lin_alg_test.exs

```diff
@@ -1,5 +1,6 @@
 defmodule Nx.LinAlgTest do
   use ExUnit.Case, async: true
+  use Complex.Kernel
```

Comment:
I don't thing we should use Complex.Kernel here

Reply:
This description is unnecessary and should be deleted.

Reply:
I don't get what you mean

Reply:
I'll delete `use Complex.Kernel`.

Reply:
I deleted `use Complex.Kernel`.